dts/msm8916-samsung.dts

```diff
@@ -23,4 +23,11 @@
 		compatible = "samsung,a5u-eur", "qcom,msm8916", "lk2nd,device";
 		lk2nd,match-bootloader = "A500FU*";
 	};
+	
+	j5xnlte-eur {
```

Comment:
Where does the `n` in `j5xnlte` come from? Is that just in the Android codename? Because in downstream, the device tree is called `msm8916-sec-j5xlte-eur-r04.dts` as far as I can tell (without `n`)